pkg/machines/components/vm/stateIcon.jsx

```diff
@@ -0,0 +1,77 @@
+/*jshint esversion: 6 */
+/*
+ * This file is part of Cockpit.
+ *
+ * Copyright (C) 2016 Red Hat, Inc.
+ *
+ * Cockpit is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 2.1 of the License, or
+ * (at your option) any later version.
+ *
+ * Cockpit is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+ */
+import cockpit from 'cockpit';
+import React, { PropTypes } from "react";
+import {
+    rephraseUI,
+} from "../../helpers.es6";
+
+const _ = cockpit.gettext;
+
+React;
+
+const StateIcon = ({ state, config, valueId, extra }) => {
+    if (state === undefined) {
+        return (<div/>);
+    }
+
+    let stateMap = {
+        running: { className: 'pficon pficon-ok icon-1x-vms', title: _("The VM is running.") }, // TODO: display VM screenshot if available or the ok-icon otherwise
+        idle: { className: 'pficon pficon-running icon-1x-vms', title: _("The VM is idle.") },
+        paused: { className: 'pficon pficon-pause icon-1x-vms', title: _("The VM is paused.") },
+        shutdown: { className: 'glyphicon glyphicon-wrench icon-1x-vms', title: _("The VM is going down.") },
+        'shut off': { className: 'fa fa-arrow-circle-o-down icon-1x-vms', title: _("The VM is down.") },
+        crashed: { className: 'pficon pficon-error-circle-o icon-1x-vms', title: _("The VM crashed.") },
+        dying: {
+            className: 'pficon pficon-warning-triangle-o icon-1x-vms',
+            title: _("The VM is in process of dying (shut down or crash is not completed)."),
+        },
+        pmsuspended: {
+            className: 'pficon pficon-ok icon-1x-vms',
+            title: _("The VM is suspended by guest power management."),
+        },
+        creating: { className: 'pficon pficon-pending icon-1x-vms', title: _("The VM is being created.") },
+        'initializing installation': {
+            className: 'pficon pficon-pending icon-1x-vms',
+            title: _("Installation of the VM is being initialized."),
```

Comment:
This is using passive voice. Changing it to active voice would make it "Initializing VM installation", a much simpler sentence. https://www.englishpage.com/verbpage/activepassive.html
If so, I think we can drop the tooltip, and just have the label say that.
I also wonder if "Starting" is a simpler and better word to use than "Initializing", but maybe there is some nuances I'm not aware of here. English is not my primary language.
The rest looks good to go to me. Thanks for working on this!

Reply:
I think "Creating" is better - it's creating the disk image. "Starting" sounds too much like "booting", and also it would contradict the "Immediately start VM" check box if this is ticked off.

Reply:
- changed to `creating VM` and `creating VM installation` 
- removed tooltip